packages/core/src/anchor/auth/did-anchor-service-auth.ts

```diff
@@ -40,7 +40,14 @@ export class DIDAnchorServiceAuth implements AnchorServiceAuth {
       throw new Error('Missing Ceramic instance required by this auth method')
     }
     const { request } = await this.signRequest({ url, opts })
-    return await this._sendRequest(request)
+    return await this._sendRequest(request).catch((err) => {
+      if (err.message.includes("status 'Unauthorized'")) {
+        throw new Error(
+          `You are not authorized to use the anchoring service found at: ${this._anchorServiceUrl}. Are you using the correct anchoring service url? If so please ensure that you have access to 3Box Labs’ Ceramic Anchor Service by following the steps found here: https://composedb.js.org/docs/0.4.x/guides/composedb-server/access-mainnet`
+        )
```

Comment:
ok stuck it in plat for now, probably this will be a next season thing https://linear.app/3boxlabs/issue/PLAT-1495/permalinks-to-doc-pages-would-be-good-for-error-messages-and-other

Reply:
thanks =)